centreon/src/Core/HostTemplate/Domain/Model/NewHostTemplate.php

```diff
@@ -128,14 +126,14 @@ public function __construct(
     ) {
         $shortName = (new \ReflectionClass($this))->getShortName();
 
-        // Assertions on string properties
-        $this->name = trim($name);
+        // Formating and assertions on string properties
+        $this->checkCommandArgs = array_map((fn($arg) => self::formatCommandArg($arg)), $checkCommandArgs);
+        $this->eventHandlerCommandArgs = array_map((fn($arg) => self::formatCommandArg($arg)), $eventHandlerCommandArgs);
```

Comment:
Superfluous parentheses + static.
```suggestion
        $this->checkCommandArgs = array_map(static fn($arg) => self::formatCommandArg($arg), $checkCommandArgs);
        $this->eventHandlerCommandArgs = array_map(static fn($arg) => self::formatCommandArg($arg), $eventHandlerCommandArgs);
```